services/route_manager/src/main/resources/application.properties

```diff
@@ -1,11 +1,14 @@
 #server.port=8082
 #spring.redis.host=redisRoute
 #spring.redis.host=redis2
-spring.redis.host=10.103.179.92
-spring.redis.port=6380
+#spring.redis.host=10.103.179.92
+#spring.redis.port=6380
 #apache.kafka.address=172.17.0.1:9092
 #Logging configuration
 logging.level.root=info
 logging.level.org.springframework.web=info
 logging.file.path=.
 logging.type=file
+#Ignite configuration
+#ignite.host=192.168.131.131
```

Comment:
Is this change tested in K8s? I think we need to add ymal files in alcor/kubernetes/services folder.

Reply:
Yeah we need to add new ignite ymal files. I haven't test ignite in kubernetes before.

Reply:
Yes, I tested it when I deployed ignite in K8s.

Reply:
I have added some yaml files to ignite and tested the ignite deployment in K8s before.

Reply:
Here are some yaml files for ignite deployment in k8s: https://github.com/futurewei-cloud/alcor/tree/master/kubernetes/db/ignite